source/content/guides/git/05-undo-commits.md

```diff
@@ -174,6 +175,10 @@ You can revert a past commit that has been pushed to your Test or Live environme
 
 You can reset your Dev environment history to match the current state of your Live environment using [Terminus](/terminus). The method in this section is destructive and should be used with caution. It does not clone the Live environment's database or files down to Dev. However, it does reset the Dev environment's codebase.
 
+<TabList>
+
+<Tab title="All others" id="all-others" active={true}>
+
 1. Identify the most recent commit deployed to Live.
```

Comment:
Actually, the Git/Terminus command does the identifying. So it seems like this is not really a "Step". I think it was added as a numbered list of steps by mistake a while back.